simulator/src/display.rs

```diff
@@ -62,17 +62,18 @@ where
     }
 }
 
-impl<C> DrawTarget<C> for SimulatorDisplay<C>
+impl<CP, CD> DrawTarget<CP> for SimulatorDisplay<CD>
```

Comment:
Minor nit, but can we change these to `PC` for Pixel Color` and `DC` for Display Color? If you'd prefer to keep them as is, please add a (non-doc) comment above with a quick explanation - the names and usages are a bit confusing even to me.

Reply:
I know they aren't great, but I'm not sure that reversing the letters will make them better. They both are colors so I decided to use a common `C` prefix for both because all other color generics use `C`.

 Should we just use `PixelColor` and `DisplayColor` instead? Longer names for generics aren't very common but I think they would make this code more readable.

Reply:
> Should we just use PixelColor and DisplayColor instead? Longer names for generics aren't very common but I think they would make this code more readable.

That crossed my mind too. In this case yeah I think that's the best option so let's do that.

Reply:
Hmm, I just noticed that `PixelColor` is obviously not a good name. Maybe `DrawableColor`?

Reply:
Or `PixelC` and `DisplayC`?

Reply:
> Hmm, I just noticed that `PixelColor` is obviously not a good name.

Oh... yeah oops

> Or `PixelC` and `DisplayC`?

Yeah that works, it fixes the ambiguities imo